.github/workflows/update-urls.yml

```diff
@@ -0,0 +1,31 @@
+name: Update URLS
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  updateURLS:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ide
+        uses: actions/checkout@v3
+      - name: Checkout ide-urls
+        uses: actions/checkout@v3
+        with:
+          repository: devonfw/ide-urls
+          path: ide-urls
+          token: ${{ secrets.ACTION_PUSH_TOKEN }}
+      - name: Build and run url updater
+        run: |
+          cd url-updater
+          mvn install
+          mvn exec:java -Dexec.mainClass="com.devonfw.tools.ide.url.UpdateInitiator" -Dexec.args="../ide-urls"
+      - name: Commit and push to ide-urls
+        run: |
+          cd ide-urls
+          git add .
+          git config --global user.name ${{ secrets.BUILD_USER }}
+          git config --global user.email ${{ secrets.BUILD_USER_EMAIL }}
+          git commit -m "Update urls"
+          git push
```

Comment:
as discussed in PR #1093 we need to solve the push problem as this will also not work as is.

Reply:
Lets give it a try with the push token...